csrankings-9.csv

```diff
@@ -1927,8 +1927,8 @@ Yves-Stan Le Cornec,Ecole Normale Superieure de Lyon,http://www.ens-lyon.fr/m-yv
 Yvo Desmedt,University of Texas at Dallas,http://www.utdallas.edu/~Yvo.Desmedt,NOSCHOLARPAGE
 Yvonne Coady,University of Victoria,http://www.cs.uvic.ca/~ycoady,Z4GOmPMAAAAJ
 Yvonne Dittrich,IT University of Copenhagen,https://pure.itu.dk/portal/en/persons/yvonne-dittrich(f2136570-f73c-446f-9fb0-6d9e461e2124).html,K6o-8eEAAAAJ
-Yvonne J. F. M. Jansen,University of Copenhagen,http://www.diku.dk/Ansatte/?pure=da%2Fpersons%2Fyvonne-jansen(b7d34108-fa8c-4a46-9796-c572d2b4f27b).html,WM1ClEoAAAAJ
-Yvonne Jansen,University of Copenhagen,http://www.diku.dk/Ansatte/?pure=da%2Fpersons%2Fyvonne-jansen(b7d34108-fa8c-4a46-9796-c572d2b4f27b).html,WM1ClEoAAAAJ
+Yvonne J. F. M. Jansen,CNRS,http://yvonnejansen.me/,WM1ClEoAAAAJ
+Yvonne Jansen,CNRS,http://yvonnejansen.me/,WM1ClEoAAAAJ

```

Comment:
CNRS is new so its entire faculty must be added (or this person removed from the database until that happens).

Reply:
CNRS is tricky because it's 15.000 people overall and I'd think likely hundreds of CS people. And man of them embedded at regular universities. I think that's beyond the scope of this pull request and I've hence removed Yvonne Jansen from the database.